.github/workflows/ScenarioTest.yaml

```diff
@@ -91,104 +91,104 @@ jobs:
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=lanechange_left map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=15 > log.txt
           cat log.txt | grep "Scenario Succeed"
 
       - name: Run merge_left scenario
         if: matrix.test_flags == 'cpp_mock_test'
         run: |
           source ~/ros2_ws/install/setup.bash
-          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=merge_left timeout:=15 > log.txt
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=merge_left map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=15 > log.txt
           cat log.txt | grep "Scenario Succeed"
 
       - name: Run phase_control scenario
         if: matrix.test_flags == 'cpp_mock_test'
         run: |
           source ~/ros2_ws/install/setup.bash
-          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=phase_control timeout:=15 > log.txt
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=phase_control osm_file:=lanelet2_map_with_private_road_and_walkway_ele_fix.osm map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=15 > log.txt
           cat log.txt | grep "Scenario Succeed"
 
       - name: Run traveled_distance scenario
         if: matrix.test_flags == 'cpp_mock_test'
         run: |
           source ~/ros2_ws/install/setup.bash
-          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=traveled_distance timeout:=15 > log.txt
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=traveled_distance osm_file:=lanelet2_map_with_private_road_and_walkway_ele_fix.osm map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=15 > log.txt
           cat log.txt | grep "Scenario Succeed"
 
       - name: Run momentary stop scenario
         if: matrix.test_flags == 'cpp_mock_test'
         run: |
           source ~/ros2_ws/install/setup.bash
-          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=momentary_stop timeout:=20 > log.txt
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=momentary_stop osm_file:=lanelet2_map_with_private_road_and_walkway_ele_fix.osm map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=20 > log.txt
           cat log.txt | grep "Scenario Succeed"
 
       - name: Run reaction_time scenario
         if: matrix.test_flags == 'cpp_mock_test'
         run: |
           source ~/ros2_ws/install/setup.bash
-          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=reaction_time timeout:=10 > log.txt
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=reaction_time osm_file:=lanelet2_map_with_private_road_and_walkway_ele_fix.osm map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=10 > log.txt
```

Comment:
.osm file should be exactly fixed in scenario. (C++ source code)
So, please remove osm_file and map_path arguments.